modules/ossm-auto-route.adoc

```diff
@@ -29,12 +33,11 @@ spec:
        ior_enabled: true
 ----
 
-For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
 
 [id="ossm-auto-route-subdomains_{context}"]
-== Subdomains	
+== Subdomains
 
-{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default. Cluster administrators can refer to xref:../../networking/ingress-operator.html#using-wildcard-routes_configuring-ingress[Using wildcard routes] for instructions on how to enable subdomains.
```

Comment:
@JStickler Why did this drop out? I'm pretty sure I put that link in there to satisfy a Jira that I can look up.

Reply:
That's one of the assemblies that I turned into a module.  See the module ossm-auto-route.adoc.   The content is still there.

Reply:
The topic is in the "Customizing installation" assembly, since enabling it involves editing the gateway definition.